### PR TITLE
Update getCourseInfo and getCoursesList queries to use graphql

### PR DESCRIPTION
**Pull request by sohumshah2**
Update getCourseInfo and getCoursesList queries to use graphql

### CODE REVIEW

File: client/src/constants/timetable.ts [hunk 1]
```diff
@@ -1,11 +1,140 @@
...
+      })
+    ) {
+      return SUPPORTED_TERMS[nextTermIdx(currTermIndex)];
+    }
+  }
+
+  return 'Invalid Term';
+};
+
+// TODO: remove
```
Comment (Rayahhhmed) [inline client/src/constants/timetable.ts]: Do we need this? how do we know what term are we currently in?

File: client/src/constants/timetable.ts [hunk 2]
```diff
@@ -1,11 +1,140 @@
...
+//       }
+//     });
+//   }
+//   console.log(termOfferingMap);
+// };
 
 const REGULAR_TERM_STR_LEN = 2;
 
 const parseYear = (termDate: string) => {
+  console.log(termDate);
```
Comment (Rayahhhmed) [inline client/src/constants/timetable.ts]: remove clg

File: client/src/api/config.ts [hunk 3]
```diff
@@ -10,10 +12,15 @@ interface Config {
   auto: string;
   server: string;
 }
-
+const HASURAGRES_GRAPHQL_API = 'https://graphql.csesoc.app/v1/graphql';
 const LOCAL = 'http://localhost:3001';
 const LIVE = 'https://timetable.devsoc.app';
```
Comment (Rayahhhmed) [inline client/src/api/config.ts]: Get rid of this, we should stop using the old timetable

**Comment by Rayahhhmed** [comment]
Theres merge conflicts too, go to master pull and merge master in to prevent them from breaking prod

**Review by Shzmj — APPROVED** [review]
Nice work!

**Review by Rayahhhmed — APPROVED** [review]

**Pull request merged**